BrainPortal/public/javascripts/cbrain.js

```diff
@@ -500,37 +435,37 @@
   $(function() {
     $("body").bind("new_content", load_behaviour);
     $("body").trigger("new_content");
-    
-    
+
+
     /////////////////////////////////////////////////////////////////////
     //
     // Ajax Pagination
     //
     /////////////////////////////////////////////////////////////////////
```

Comment:
Please make sure this is still in used. AFAIK most `.page_links` elements have been replaced by dynamic table pagination elements.